src/getCoinCombination.test.js

```diff
@@ -9,12 +9,30 @@ describe(`Function 'getCoinCombination':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(getCoinCombination(1)).toBeInstanceOf(Array);
```

Comment:
You do not need here parameter. Just empty function().

Reply:
Why? You can pass any parameters to the function, so both options are fine

Reply:
thank God)